Arrowgene.Ddon.Shared/Network/ServerType.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace Arrowgene.Ddon.Shared.Network
 {
-    public enum ServerType
+    public enum ServerType : ushort
     {
-        Login,
-        Game
+        Login = 52100,
+        Game = 52000
```

Comment:
This is fine for packet parsing however im not too sure about it being in such a global enum, used in other places, where the ports can be changed. I feel like you should map to ServerType from port in the command class

Reply:
Totally overlooked that. A lot of custom code/classes exist for the packet command, so I didn't see it's part of the Network project.
Will take care of it.

Reply:
Opted for duplicating & adapting a special packet-only ServerType.